api/api_full.go

```diff
@@ -712,16 +711,17 @@ type MinerSectors struct {
 
 type ImportRes struct {
 	Root     cid.Cid
-	ImportID multistore.StoreID
+	ImportID uint64
```

Comment:
Would be nice to have this still be a type, makes it more clear which things are what in all the places

Reply:
Have fixed this by introducing a new `ImportID` type.